src/util.h

```diff
@@ -663,6 +663,12 @@ void replace_in_buffer(MemoryRange src, const uint8_t* src_data,
 
 // Strip any directory part from the filename `s`
 void base_name(std::string& s);
+struct TraceOutputPath {
+  std::string output_trace_dir;
```

Comment:
Please carefully document these fields.

Reply:
See new comments added to the type - are they enough?